README.md

```diff
@@ -52,29 +82,32 @@ Once pnpm is installed, in the monorepo root run:
 pnpm install
 ```
 
-This will install the dependencies for all packages.
+This will install the dependencies for all the monorepo packages and apps.
 
-In order to run dialtone locally, you can use:
+### Running the projects
+
+#### Dialtone
 
 ```bash
-# This will start the server for the documentation site and the library so it is live updated with any changes.
-pnpm run start:dialtone
+nx start:dialtone
```

Comment:
I don't have `nx` installed globally and it's working well for me, it was failing on CI only, is it failing for you?

Reply:
I had to install it globally to make it work too

Reply:
You probably installed nx globally at some point while you were working on this. We should probably add the command to add nx globally to the readme. I think it is a bit less confusing to run `nx [command]` rather than `pnpm nx [command]`

Reply:
Added, maybe while first initializing the monorepo it did actually installed something on my machine, add docs related to global NX.